payment-handler/change-payment-method.https.html

```diff
@@ -0,0 +1,122 @@
+<!doctype html>
```

Comment:
Just a minor thing, I'd suggest using https://prettier.io to tidy up the code.

Reply:
Used this command:

```bash
yarn prettier \
  --single-quote \
  --trailing-comma es5 \
  --no-bracket-spacing \
  --jsx-bracket-same-line \
  --arrow-parens always \
  --write <file-name> 
```